crates/core/tedge_mapper/src/c8y/mapper.rs

```diff
@@ -114,6 +114,7 @@ fn create_directories(config_dir: &str) -> Result<(), anyhow::Error> {
 }
 
 #[cfg(test)]
+#[cfg(feature = "integration-test")]
```

Comment:
Why a new feature flag for all these tests? When these tests will be run in the CI/CD pipeline?

Reply:
these tests now need:
- tedge config access to know where the logs path is
- some tests also need actual logs available in the logs path

these lines have been added to `CumulocityConverter::new` function. 

So all the tests that require the mapper now also require a logs path to exist. 
```rust
        ...
        // inside CumulocityConverter::new
        let tedge_config = get_tedge_config()?;
        let logs_path = tedge_config.query(LogPathSetting)?;

        let log_dir = PathBuf::from(&format!("{}/{TEDGE_LOG}", logs_path));

        let operation_logs = OperationLogs::try_new(log_dir)?;
```

Reply:
> When these tests will be run in the CI/CD pipeline?

they will run in the integration workflow, after merging